pvlib/test/test_tools.py

```diff
@@ -1,6 +1,10 @@
+import numpy as np
+import pandas as pd
 import pytest
 
+import pvlib.tools
 from pvlib import tools
+from pvlib.test.test_solarposition import times
```

Comment:
Please create an appropriate new input within the test function below. Dependencies across test modules are difficult to maintain.